apps/vr-tests-react-components/package.json

```diff
@@ -74,6 +73,7 @@
     "@griffel/react": "^1.5.14",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "tslib": "^2.1.0"
+    "tslib": "^2.1.0",
+    "@fluentui/react-breadcrumb": ">=9.0.0-alpha"
```

Comment:
should this be "*" like the other packages in this list?

Reply:
No, because it needs a lower version than 9.0.0 to use as a reference to bump to the next